pubspec.yaml

```diff
@@ -13,6 +13,7 @@ dependencies:
   http: ^0.13.0
   fluttertoast: ^8.0.8
   flutter_inappwebview: ^5.3.2
+  webview_flutter: ^3.0.4
```

Comment:
Is there a reason why this was added, @Utitofon-Udoekong ?

Reply:
Looking at the webview.dart file in the view folder, the plugin was used, but it wasn't added in the pubspec.yaml.